src/gradlew

```diff
@@ -1,25 +1,43 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
Not sure if it's a good idea to modify this file since it's a `gradle` thing.

Reply:
It doesn't affect anything in itself. It's merely using git bash and running "./gradlew wrapper --gradle-version 4.10.2 --distribution-type all", basically keeping all gradle(w) files at the correct version with each other.